SeQuant/domain/mbpt/convention.cpp

```diff
@@ -72,23 +72,49 @@ void register_standard_instances() {
     auto declab = [s](auto&& label) {
       return qndecorate(static_cast<qns>(s), label);
     };
+    // these spaces are used in Fock space methods
+    // based on single-determinant references
+    // p,q,r... for OBS spstates introduced in DOI 10.1063/1.444231 (QCiFS I)
+    IndexSpace::register_instance(declab(L"p"), IndexSpace::all, qnattr,
+                                  do_not_throw);
+    // {i,j,k.../a,b,c...} for active {occupied/unoccupied} spstates introduced
+    // in DOI 10.1063/1.446736 (QCiFS III)
     IndexSpace::register_instance(declab(L"i"), IndexSpace::active_occupied,
                                   qnattr, do_not_throw);
-    IndexSpace::register_instance(declab(L"m"), IndexSpace::occupied, qnattr,
-                                  do_not_throw);
     IndexSpace::register_instance(declab(L"a"), IndexSpace::active_unoccupied,
                                   qnattr, do_not_throw);
+    // {α,β.../κ,𝛌...} for complete {unoccupied/any} spstates introduced in
+    // DOI 10.1063/1.459921 (MP2-R12 I)
+    IndexSpace::register_instance(declab(L"α"), IndexSpace::complete_unoccupied,
+                                  qnattr, do_not_throw);
+    IndexSpace::register_instance(declab(L"κ"), IndexSpace::complete, qnattr,
+                                  do_not_throw);
+    // for orthogonal complement to p introduced in
+    // DOI 10.1016/j.cplett.2004.07.061 (CABS)
+    IndexSpace::register_instance(declab(L"α'"), IndexSpace::other_unoccupied,
+                                  qnattr, do_not_throw);
+    // m,n... for all occupied (including inactive/frozen orbitals) de facto
+    // introduced in [DOI 10.1016/j.cplett.2004.07.061
+    // (CABS)](https://dx.doi.org/10.1016/j.cplett.2004.07.061), though formally
+    // not explicitly defined so
+    IndexSpace::register_instance(declab(L"m"), IndexSpace::occupied, qnattr,
+                                  do_not_throw);
+    // introduced in MPQC LCAOWavefunction
     IndexSpace::register_instance(declab(L"e"), IndexSpace::unoccupied, qnattr,
                                   do_not_throw);
+    // introduced in MPQC for GF, CT-F12, and other ad hoc uses
     IndexSpace::register_instance(declab(L"x"), IndexSpace::all_active, qnattr,
                                   do_not_throw);
-    IndexSpace::register_instance(declab(L"p"), IndexSpace::all, qnattr,
+    // e.g. see DOI 10.1063/5.0067511
```

Comment:
@Krzmbrzl do you know the origin of this convention?

Reply:
As far as I know we have borrowed our notation mostly from internally-contracted CI theory but other than that, I don't know.
@ak-ustutt might know more though

Reply:
u, v, w, ....  or t, u, v, w, ... is rather common throughout MR works; u should be great here